test/s3select_test.cpp

```diff
@@ -1241,25 +1250,26 @@ TEST(TestS3selectFunctions, in_expressions)
 
   ASSERT_NE(s3select_result_9,failure_sign);
 
-  const std::string input_query_10 = "select int(_1) from s3object where _1 like \"_3\";";
```

Comment:
in this type of query, one option is to manually create the converted-JSON-query and pass it to `run_s3select`